server/models/notifications.js

```diff
@@ -0,0 +1,54 @@
+const getNotificationsModel = (sequelize, DataTypes) => {
+  const flags = {
+    freezeTableName: true
+  };
+
+  const NotificationShema = {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      allowNull: false,
+      defaultValue: DataTypes.UUIDV4,
+      unique: true
+    },
+    userId: {
+      type: DataTypes.UUID
+    },
+    message: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    isSeen: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
+      allowNull: false
+    },
+    source: {
+      type: DataTypes.UUID
+    }
+  };
+
+  const Notification = sequelize.define(
+    "Notifications",
+    NotificationShema,
+    flags
+  );
+
+  Notification.associate = db => {
+    Notification.belongsTo(db["Users"], {
+      foreignKey: "userId",
+      target: "id",
+      onDelete: "cascade"
```

Comment:
Change cascade to CASCADE in all the files for the sake of consistency in the code base.